presto-main/src/main/java/com/facebook/presto/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -307,6 +308,17 @@ public static Block regexpSplit(@SqlType("varchar(x)") Slice source, @SqlType(Jo
         return blockBuilder.build();
     }
 
+    private static int getMatchingOffset(Matcher matcher, int at, int range, int option)
```

Comment:
We can remove `option` in the param and put `Option.DEFAULT` directly in the body.

Reply:
This commit is going to be landed by @kaikalur in https://github.com/prestodb/presto/pull/16109 
I'll ask for it there.